### ISSUE TITLE
How to see performance values in training log

### ISSUE THREAD

**Issue by liyunqianggyn** [THU-MIG/yolov10#319]
Thanks for your nice work. When I am looking into training log values for mAP, precision..., I find a weird pattern for these values that consecutive 10 epochs have exactly same values (see below fig) even they have different training learning rate/loss, and this happens during whole training, could you explain the weird phenomenons, thanks!  
<img width="527" alt="image" src="https://github.com/THU-MIG/yolov10/assets/29226200/aef77cc9-a8fa-4499-aedd-b5586412fae0">
